docs/causal_graph_utils.md

```diff
@@ -141,3 +141,33 @@ skeleton: Skeleton = cg.skeleton
 # note the order may not match but the elements will be those four.
 markov_boundary: List[str] = identify_markov_boundary(skeleton, node='a')
 ```
+
+### Identifying Colliders
```

Comment:
Would put this above Markov Boundary one. Think it flows better going after mediators and stuff.